commands/law.js

```diff
@@ -0,0 +1,20 @@
+import Discord from "discord.js";
+
+export const command = {
+  name: "law",
+  category: "info",
+  description: "Shows the buddy law.",
+  run: async (client, message) => {
+    const embed = new Discord.MessageEmbed()
+      .setColor("#0099ff")
+      .setTitle("The Buddy Law")
+      .setDescription(
+        "1) A buddy should be kind, helpful and trustworthy to each other!\n" +
+          "2) A buddy must be always ready for anything!\n" +
+          "3) A buddy should always show a bright smile on his face!\n" +
+          "||4) We leave no buddy behind!||"
+      );
+
+    await message.channel.send(embed);
```

Comment:
Why are you awaiting this?
```suggestion
    message.channel.send(embed);
```